setup.py

```diff
@@ -1,10 +1,16 @@
+# noqa: D100
 import os
 
 from setuptools import setup
 
 here = os.path.abspath(os.path.dirname(__file__))
 README = open(os.path.join(here, 'README.rst')).read()
 
+requirements = None
```

Comment:
Why do want to externalise the requirements?

Reply:
I thought this made it easier to install with `pip install -r`. maybe I'm mistaken?...

Reply:
This is a module which you install with *pip install <path>* if you are installing it locally, or just by name when pulling it remotely or *pip install -e .* if you want to install an editable version for development purposes.
There is no need for an aexternal requirements file

Reply:
ok, no problem, I can revert that change then.

I was wondering about the python packages installed as a hardcoded list in the `.travis.yaml` file: those might be interesting to keep around in a file that's easier access to ppl doing dev work on the library. We could add them to the `tests_require=` parameter to the `setup()` function in setup.py but then I'm not sure if it's usable in some form with pip install.

Reply:
ah .. hmm. after posting my last comment, I searched a bit more info on that option and found a comment about it being deprecated since version 41.5.0 of setuptools. I guess it would correspond to the line mentioning tha the `test` command is deprecated.

https://setuptools.readthedocs.io/en/latest/history.html#v41-5-0

however there's still been some changes about `tests_require` in later version releases.. so now I'm confused. maybe I should just not bother with this at all for now and it can be figured out at another time and in another issue/PR.

Reply:
The `test` command was (unfortunately - it had it's advantages) deprecated. Without it, `test_requires` does not make sense anymore. You could create a `dev_requirements.txt` for such a purpose for example.